admin-dev/themes/new-theme/js/pages/cart/index.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * vCopyright since 2007 PrestaShop SA and Contributors
```

Comment:
```suggestion
 * Copyright since 2007 PrestaShop SA and Contributors
```
Little typo, I guess a copy-paste missclick 😄